common/determined_common/api/experiment.py

```diff
@@ -17,6 +21,137 @@ def activate_experiment(master_url: str, exp_id: int) -> None:
     patch_experiment(master_url, exp_id, {"state": "ACTIVE"})
 
 
+def logs(args: Namespace) -> None:
```

Comment:
blocking: When you add `logs`, probably be better to also remove the `logs` function in `cli/determined_cli/trial` otherwise we are duplicating one function in two different places, which is bad.